CMakeLists.txt

```diff
@@ -20,19 +20,20 @@ if (DEBUG)
   add_definitions(-DDEBUG)
 endif ()
 
-
 aux_source_directory(${PROJECT_SOURCE_DIR}/src/aligner ALIGNER_SRCS)
 add_library(libaligner ${ALIGNER_SRCS})
 include_directories(${PROJECT_SOURCE_DIR}/src/aligner)
 
+aux_source_directory(${PROJECT_SOURCE_DIR}/src/similaritymatrix SIMILARITY_MATRIX_SRCS)
+add_library(libsimilaritymatrix ${SIMILARITY_MATRIX_SRCS})
```

Comment:
sources in /src/similaritymatrix could be put into /src/aligner

Reply:
Makes sense.

Reply:
Simply build everything as libaligner and it would be all fine

Reply:
just `include_directories(${PROJECT_SOURCE_DIR}/src/aligner/similaritymatrix)`?

Reply:
Like this
```
/src/aligner/
 - LocalAligner.h
 - SWAligner.h
 - SWAligner.cpp
 - similarity_matrix.h
 - similarity_matrix.cpp
```
and only keep 
```
aux_source_directory(${PROJECT_SOURCE_DIR}/src/aligner ALIGNER_SRCS)
add_library(libaligner ${ALIGNER_SRCS})
include_directories(${PROJECT_SOURCE_DIR}/src/aligner)
```
in `CMakeFiles.txt`.